cla_backend/libs/eligibility_calculator/cfe_civil/savings.py

```diff
@@ -1,25 +1,39 @@
 from cla_backend.libs.eligibility_calculator.cfe_civil.conversions import pence_to_pounds, none_filter
 
 
-def _savings_value(value, description):
-    if value > 0:
+def _savings_value(savings_data, attr, description):
+    if hasattr(savings_data, attr):
         return {
-            "value": pence_to_pounds(value),
+            "value": pence_to_pounds(getattr(savings_data, attr)),
             "description": description,
             "subject_matter_of_dispute": False
         }
 
 
+_CFE_SAVINGS_KEY = "capitals"
+
+
+def has_savings_key(request_data):
+    return _CFE_SAVINGS_KEY in request_data
+
+
 def translate_savings(savings_data):
-    liquid_capital = [
-        _savings_value(savings_data.bank_balance, "Savings"),
-        _savings_value(savings_data.investment_balance, "Investment")
+    liquid = [
+        _savings_value(savings_data, "bank_balance", "Savings"),
+        _savings_value(savings_data, "investment_balance", "Investment")
     ]
-    non_liquid_capital = [
-        _savings_value(savings_data.asset_balance, "Asset")
+    non_liquid = [
+        _savings_value(savings_data, "asset_balance", "Asset")
     ]
+    liquid_capital = none_filter(liquid)
+    non_liquid_capital = none_filter(non_liquid)
 
-    return dict(capitals={
-        "bank_accounts": none_filter(liquid_capital),
-        "non_liquid_capital": none_filter(non_liquid_capital)
-    })
+    if len(liquid_capital) + len(non_liquid_capital) < 3:
```

Comment:
3 seems very specific, and a source of bugs if we change the number of fields to expect. Either we need a better condition to figure, or simply return `{"back_accounts": [], "non_liquid_capital": []}`